crates/aptos-drop-helper/src/async_concurrent_dropper.rs

```diff
@@ -1,10 +1,9 @@
 // Copyright © Aptos Foundation
 // SPDX-License-Identifier: Apache-2.0
 
-use crate::{
-    metrics::{GAUGE, TIMER},
-    IN_ANY_DROP_POOL,
-};
+#[cfg(not(feature = "no-metrics"))]
```

Comment:
With positive features, this will stop being a brain-twister:

```suggestion
#[cfg(feature = "metrics")]
```

Reply:
That's not a brain twister. That's how folks speak proper Southern English.